common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1649,6 +1655,39 @@
     ],
     "sqlState" : "22000"
   },
+  "HYBRID_ANALYZER_EXCEPTION" : {
+    "message" : [
+      "An failure occurred when attempting to resolve a query or command with both the legacy fixed-point analyzer as well as the single-pass resolver."
+    ],
+    "subClass" : {
+      "FIXED_POINT_FAILED_SINGLE_PASS_SUCCEEDED" : {
+        "message" : [
+          "Fixed-point resolution failed, but single-pass resolution succeeded.",
+          "Single-pass analyzer output:",
+          "<singlePassOutput>"
+        ]
+      },
+      "LOGICAL_PLAN_COMPARISON_MISMATCH" : {
+        "message" : [
+          "Outputs of fixed-point and single-pass analyzers do not match.",
+          "Fixed-point analyzer output:",
+          "<fixedPointOutput>",
+          "Single-pass analyzer output:",
+          "<singlePassOutput>"
+        ]
+      },
+      "OUTPUT_SCHEMA_COMPARISON_MISMATCH" : {
+        "message" : [
+          "Output schemas of fixed-point and single-pass analyzers do not match.",
+          "Fixed-point analyzer output schema:",
+          "<fixedPointOutputSchema>",
+          "Single-pass analyzer output schema:",
+          "<singlePassOutputSchema>"
+        ]
+      }
+    },
+    "sqlState" : "0A000"
```

Comment:
The class `0A` is related to unsupported feature, see `error-classes.json`. Let's assign another value.

Reply:
I'm looking at the [existing codes](https://spark.apache.org/docs/3.5.3/sql-error-conditions-sqlstates.html) and I cannot find the suitable one. Those errors are related to the cases when single-pass and fixed-point results don't match.

Reply:
Can we treat such errors as internals in the class:
```
    "XX": "Internal Error"
```
?

Reply:
Yeah, this is purely internal and is never thrown to the user in normal mode of operation (only for Analyzer dual runs).